reuse/contribute-to-docs.txt

```diff
@@ -28,18 +28,101 @@ Use the :guilabel:`Give feedback` button at the top of any page to create a GitH
+
+.. code:: rst
+
+   .. toctree::
+      :maxdepth: 2
+
+      subcategory-one/index
+      Subcategory two <subcategory-two/index>
+      page-one-file-name
+
+For more information, read the `Sphinx documentation on toctree`_.
+
+5. Update the documentation base ``index.rst`` file by adding your new category to the toctree.
+
+Create new subcategories
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To add a subcategory within an existing category in your documentation, follow these steps:
+
+1. Go to the parent category and create a new folder for your subcategory within it.
+2. Create an ``index.rst`` file within the subcategory folder.
+3. Enter the title of your new subcategory on the first line of the ``index.rst`` file. Underline it using equal signs (=) that match the length of your title. For more information on titles and headings, read the `reStructuredText`_ style guide.
+4. In the ``index.rst`` file, add content introducing the subcategory, its purpose, and other relevant links.
+5. In your ``index.rst`` file, add a toctree that includes the file names or titles of pages within your new subcategory. The toctree should resemble the following structure:
+
+.. code:: rst
+
+   .. toctree::
+      :maxdepth: 1
+
+      page-one-file-name
+      Page Two Title <page-two-file-name>
+
+.. note::
+
+   Ensure to replace `page-one-file-name` and `page-two-file-name` with the actual file names of your pages and `Page Two Title` with the actual file title. For more information, read the `Sphinx documentation on toctree`_.
+
+6. Update the ``index.rst`` file of the parent category by adding a reference to the newly created subcategory in its toctree.
+
+Create new pages
+~~~~~~~~~~~~~~~~~~
+
+You can create new pages for existing documentation by following these steps:
+
+1. Create a new file within a category or subcategory.
+#. Add a title to the first line of the file. Underline it using equal signs (=) that match the length of your title. For more information on titles and headings, read the `reStructuredText`_ style guide.
+#. Add content to the new file using `reStructuredText`_ and following the `Canonical style guide`_.
+#. Update the category or subcategory's ``index.rst`` file by adding the file name or your preferred title to the toctree. For more information, read the `Sphinx documentation on toctree`_.
 
 Download and install the docs
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-If you are working with these docs for the first time, you'll need to create a fork of the `ubuntu-cloud-docs`_ repository on your GitHub account and then clone that fork to your local machine. Once cloned, go into the ubuntu-cloud-docs directory and run:
+If you are working with these documentations for the first time, you have to create a fork of the `ubuntu-cloud-docs`_ repository on your GitHub account and then clone that fork to your local machine. Once cloned, go into the ubuntu-cloud-docs directory and run:
```

Comment:
We try to have a relatively easy-going tone, rather than being too formal / prescriptive. So I don't think these changes are essential.